mycli/main.py

```diff
@@ -425,6 +425,13 @@ def prompt_tokens(cli):
                         end = time()
                         total += end - start
                         mutating = mutating or is_mutating(status)
+                except UnicodeDecodeError:
+                    import pymysql
+                    ver = pymysql.__version__.split('.')
+                    if ver < ('0', '6', '7'):
+                        message = ('You are running an older version of pymysql.\n'
+                                'Please upgrade to 0.6.7 or above to view binary data.')
+                        self.output(message)
```

Comment:
What if an `UnicodeDecodeError` is raised, but the PyMySQL version is 0.6.7 or above? Shouldn't it raise the original error message?